src/Symfony/Component/Validator/Constraints/Expression.php

```diff
@@ -54,12 +54,4 @@ public function getTargets()
     {
         return array(self::CLASS_CONSTRAINT, self::PROPERTY_CONSTRAINT);
     }
-
-    /**
-     * {@inheritdoc}
-     */
-    public function validatedBy()
-    {
-        return 'validator.expression';
```

Comment:
Even if we add the possibility for multiple aliases and give the service both of them? I'm not sure how that works.